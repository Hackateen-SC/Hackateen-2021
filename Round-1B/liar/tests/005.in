500
160661908 564209945 311385585 38615427 541139464 78134433 873457869 707763592 376067691 802149906 156206942 837124797 441273813 330373734 922402777 499483187 848424619 949282589 571792452 755441332 255859432 871524177 838031807 630622902 290004063 369893534 734660401 929902160 466558191 634694607 162627059 613722370 549306283 578614077 831444795 826295266 424435318 651593049 510265229 569155376 613527766 357592324 67055080 77207540 485628982 887346990 481027321 71205940 492027536 233646138 38868467 171167918 573148395 449932404 911082396 508474980 614346123 882841801 957550353 39176032 7119093 52368180 85650808 216215921 975817406 964259568 249610559 852488748 506931122 812162725 129740892 218396937 881472207 674518700 217539679 791289456 831779062 933333742 392983567 454949849 157529211 128056710 809615070 536428900 323386258 207497286 81375164 695094067 581043073 538069738 575690182 18115302 375609499 759141883 935484449 284259527 906880367 543867259 944962637 232971395 147993748 800211873 918429780 795788508 576669440 16085209 504814750 343229167 344854118 878760157 100215336 439835351 609573486 825610575 191654768 414260751 23544880 319302918 106533421 775988079 697833507 112746202 828298719 63244483 625706825 742380270 615003995 532816646 440263975 880226078 41280036 755398499 690428815 617700868 426035751 792324495 359737711 792511316 620092765 123656793 47149792 915356144 237864844 35578257 31195780 725868160 91943042 965075530 584093624 894597266 574825920 979424270 616283325 544063328 845418088 109570697 629476641 543651558 314728115 209487689 645084152 943544726 156755744 58230539 854805868 245125044 177295557 77276617 588975847 672152845 369103823 444267527 744775788 694163091 160457730 357093979 174172765 456587721 336277349 510911633 154636712 822784361 673183732 67837006 659319823 935578125 744506623 839271068 142479938 732549181 463278115 783075208 129238855 937235035 714724594 79965711 282902180 33047440 931184511 290802540 507616990 518449530 916994774 44327058 149936753 516106421 310971474 779810433 986113404 44646328 713059349 452540561 609691916 144314195 928579791 518651331 969837104 728053875 984231437 625694704 231015508 511675875 899176891 519856458 46543493 281627896 284504465 333778525 501262725 987569469 303367643 332051914 487214290 270985103 932948539 383183646 311777856 889727329 390156430 227388275 176619571 905341239 894147289 679708243 617584538 499261181 932040418 582194833 134157923 686327827 262174592 85345637 922561396 482611825 779129487 162164847 94555550 321109014 642459844 985388403 884401872 627364042 646888368 32412178 359632335 847449391 135193867 952150182 482714319 187734184 781710686 904833863 104181980 614536861 484971747 814319218 601266832 257744908 360992073 169071103 790796896 724745836 231037930 695915518 805452337 344528541 637441559 836504834 109324185 95641477 421563190 729711400 444169135 885037384 266811029 328647971 672090470 337186736 136052264 522644737 733901623 483236158 758305343 626705969 937949836 904963360 194013358 34457509 149349650 40974601 436131831 432495512 233304110 665190575 835784328 381757642 642588151 587189022 521942162 782722302 672955033 960332016 4006319 171760207 613990419 683834783 704248674 507813059 995598088 643946740 292236422 880451232 179719074 736137842 996050926 126115842 801265707 1047560 416409872 176333295 284489515 444039412 957417103 486081510 214328600 832270766 293673112 199223801 347269582 478963195 569620191 136616334 256571699 815940330 550247086 755154196 537226396 689522651 454448618 86737359 971250635 990550376 798441914 900124848 322828260 156837369 111373272 992525977 423745674 982183635 72480194 242413334 864725499 849812764 517942610 911033890 334412611 883888864 459716240 252901592 739743649 969238502 192893769 109069630 138375563 93643571 96323081 809154123 310236388 770144797 671174500 987076882 460755419 691237146 790436183 613374493 300434994 907674140 629548292 850897831 583113312 111702563 120989231 21205161 468712055 238775242 635506284 669384848 550936617 808527087 911445569 784272153 393517908 885790779 315337082 680702902 387986014 470634562 420877063 279826666 7278697 240985486 135328775 189045089 809672162 219887528 914017736 667397470 937659315 545394183 142532145 267863406 189605459 874921608 637382772 989130351 562687054 518708756 154413864 689749151 146726020 655666515 241200147 699901504 541565648 620457136 836972643 490602566 647926613 252668084 868198957 804545076 753332879 289834782 409931476 589974099 167870554 343396945 955545559 605204460 497848001 81043012 25760229 716309950 32806106 450668474 392228561 971940362 552092322 495075986 344285838 238376841 877891741 175109024 583972177 801218769 230754810 969108316 701680498 137254661 355163768 709613374 269161547 85990049 907316798 221058316 432327803 61155848 614837594 959535033 375685994 378760369 571648583 282887203 203362021 175768804 136525501 59285161 966528901 427619454
GGGGLGGLGLGLGLGGLGGGLGLGGLLGGLGLLLGLLGGGLGLGLLLGLLLGLGGGLLLGLGGLGLGGLLLLGGGLLLLGGGLGLLGGGLLGLLGLLLLLLGLLLLGGGGGLLGGLLLGGGLLLLGLGLGGLGGLLGLLLGLGLLGLGLGLGGLLLGLLLGGLLLLLLGLLGGLGGLGGGGLLGGGLLGLLGLLLGGLGGGLLLLGGGLGLLGLGGGGGGGLLGGLLLLGGGGLGGLGGLGGGLGGGGGGGGGGLGGLGGGGLGLGLGLGGGLGGLGLLLGLGGGGLLGLGLGLLGLLGGGLGGGGGLGGGGLLGLLGLLGGGGLGGGGGGGGGGGLLLLLGGGGLLGGGLLLLGGLGGLGLLLGGGGGGGGGGLLLLGLGLGLGLLLGLLGLLGGLLLGGGLGGGGLGLGLGGGGLGGLGGGGLLLGGLGLLGGGGLGLGLGLGLGLLLGLGLLGGLGGGGGGLGGLLGGLLLGGLGLLLGLLLLLGGLLLGLGLLLLL
