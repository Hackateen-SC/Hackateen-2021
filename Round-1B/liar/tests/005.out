996050927 233
